duckduckgo.pot

```diff
@@ -4842,7 +4842,7 @@ msgstr ""
 
 # smartling.placeholder_format=C
 msgctxt "settings"
-msgid "Seach Visibility"
+msgid "Search Visibility"
```

Comment:
Sorry, I'm reverting this to avoid conflicts because we've removed this entire string in another PR. Thanks!

```suggestion
msgid "Seach Visibility"
```